reference/channels.v3.yml

```diff
@@ -1222,42 +1226,42 @@ components:
           - 'facebook by meta,amazon'
     date_created:
       name: date_created
-      description: 'Filter items by date_created. For example, `date_created=2019-09-04T00:00:00`, `date_created=2019-09-04`, or `date_created=1567573200`'
+      description: 'Filter items by date created. For example, `date_created=2019-09-04T00:00:00`, `date_created=2019-09-04`, or `date_created=1567573200`.'
       in: query
       required: false
       schema:
         type: string
     date_created_min:
       name: 'date_created:min'
-      description: 'Filter items by minimum date_created. For example, `date_created:min=2019-09-04T00:00:00`, `date_created:min=2019-09-04`, or `date_created:min=1567573200`'
+      description: 'Filter items by minimum date created. For example, `date_created:min=2019-09-04T00:00:00`, `date_created:min=2019-09-04`, or `date_created:min=1567573200`. Returns metafields created after this date.'
       in: query
       required: false
       schema:
         type: string
     date_created_max:
       name: 'date_created:max'
-      description: 'Filter items by maximum date_created. For example, `date_created:max=2019-09-04T00:00:00`, `date_created:max=2019-09-04`, or `date_created:max=1567573200`'
+      description: 'Filter items by maximum date created. For example, `date_created:max=2019-09-04T00:00:00`, `date_created:max=2019-09-04`, or `date_created:max=1567573200`. Returns metafields created before this date.'
       in: query
       required: false
       schema:
         type: string
     date_modified:
       name: date_modified
-      description: 'Filter items by date_modified. For example, `date_modified=2019-09-04T00:00:00`, `date_modified=2019-09-04`, or `date_modified=1567573200`'
+      description: 'Filter items by date modified. For example, `date_modified=2019-09-04T00:00:00`, `date_modified=2019-09-04`, or `date_modified=1567573200`.'
```

Comment:
we don't allow the epoch seconds

Reply:
@bc-rafsierk This example format is the same one I used for the others. Should I remove `date_modified=2019-09-04T00:00:00` from only this one or all of them?

Reply:
all of them added in this task 🙏

Reply:
@bc-rafsierk I went back to search for `date_modified=1567573200` and found this example in two files. I removed them. Also while I was in the files. I discovered many examples contained a duplicate example format so I removed one of them. I think we should be good now.